nwb_conversion_tools/utils/nwbephyswriterdatachunkiterator.py

```diff
@@ -1,15 +1,27 @@
 """Authors: Saksham Sharda."""
-from typing import Tuple, Iterable
 import numpy as np
+from typing import Tuple, Iterable
+from distutils.version import StrictVersion
+
 from .genericdatachunkiterator import GenericDataChunkIterator
 
+try:
+    import spikeinterface as si
+
+    if StrictVersion(si.__version__[:4]) >= StrictVersion("0.90"):
+        HAVE_SI_090 = True
+    else:
+        HAVE_SI_090 = False
+except ImportError:
+    HAVE_SI_090 = False
+
 
 class NwbEphysWriterDataChunkIterator(GenericDataChunkIterator):
-    """DataChunkIterator specifically for use on RecordingExtractor objects."""
+    """DataChunkIterator for use on RecordingExtractor (spikeinterface/spikeinterface) objects."""
 
     def __init__(
         self,
-        ephys_writer,
+        ephys_writer: si.BaseRecording,
```

Comment:
Ben also proposed renaming this more appropriately to `ephys_reader`, what do you guys thing?